lib/icinga/macroprocessor.cpp

```diff
@@ -172,7 +184,7 @@ Value MacroProcessor::EvaluateFunction(const Function::Ptr& func, const Resolver
 	Dictionary::Ptr resolvers_this = new Dictionary();
 
 	for (const ResolverSpec& resolver : resolvers) {
-		resolvers_this->Set(resolver.first, resolver.second);
+		resolvers_this->Set(resolver.Name, resolver.Obj);
 	}
```

Comment:
What's the effect of the `icinga` resolver being removed here (and for the `env` resolver not being added)?